.pre-commit-config.yaml

```
@@ -1,17 +1,43 @@
 repos:
+  - repo: https://github.com/pagopa/dx
+    rev: pre_commit_scripts@0.0.1
+    hooks:
+      - id: lock_modules
+        exclude: ^.*/(_modules|modules|\.terraform)(/.*)?$
+        # Configure the paths to your Terraform configurations
+        files: infra/resources/prod
```

Shall we check also "accessory" terraform configurations?

```suggestion
        files: infra/(resources/prod|github-runner/prod/westeurope|github_runner|identity/prod|repository)
```

---

we should but other configs will be deleted in these days except for `repository`